src/library_mem_ops.js

```diff
@@ -0,0 +1,71 @@
+var LibraryMemOps = {
```

Comment:
Could this be `library_safe_heap.js`? Or `library_getvalue.js`? MemOps makes it read like a file that could end up piling all kinds of general (unrelated) things into it.

Reply:
I tried calling it `safe_heap` but that didn't make much sense since these utilities are available even without `SAFE_HEAP`.   I could call its `library_getvalue_setvalue.js` but that seems way to long winded and specific.

I settled on `mem_ops` because `getValue` and `setValue` are our runtime utilities for manipulating memory... getting and setting values.   It makes sense to me anyway. 

Maybe `library_memory_get_set.js`?  But that seems worse to me.   What exactly don't you like about `library_mem_ops.js`?   If other utilities related to memory manipulation were to be added here wouldn't that actually make sense?  (rather than adding them to library.js?)  Why not group memory manipulation utilities together?

Reply:
I think it could be named `library_getvalue.js` even if it had both `getValue` and `setValue` in it.

The name `library_mem_ops.js` reads to me like something fundamental, foundational or general. But both `getValue` and `setValue` are functions that people should practically never be using. So my worry here was that people reading `library_mem_ops.js` might look into it thinking "oh I see, this is what I use for interacting with memory from JS", even though these functions should almost never be used (only make sense in a dynamic type dispatch problem situations).

That is why I was thinking to find a name that would be more specific in the term as opposed to being a general sounding name.

Maybe the concern is not that warranted, I don't feel too strongly. Maybe it might be good to have a comment before these functions saying something like "// Avoid using these functions, as they are orders of magnitude slower than directly accessing the heap via HEAP[U/F]8/16/32/64 arrays directly."

Reply:
OK I went with `library_getvalue.js`